src/specs/components/footer-standardised/demo/data--ec.js

```diff
@@ -1,239 +1,238 @@
-        },
-      },
-      section_id: '5',
-      type: 'corporate_name',
-    },
-    {
-      links: [
-        {
-          link: {
-            label: 'Contact the European Commission',
-            path: '/example',
-            aria_label: 'Link to Contact the European Commission',
-          },
-        },
-        {
-          link: {
-            label: 'Follow the European Commission on social media',
-            path: '/example',
-            aria_label:
-              'Link to Follow the European Commission on social media',
-          },
-        },
+    ],
+    [
+      [
         {
-          link: {
-            label: 'Resources for partners',
-            path: '/example',
-          },
+          links: [
+            {
+              link: {
+                label: 'Class name 1',
+                path: '/example',
+                aria_label: 'Link to Class name 1',
+              },
+            },
+            {
+              link: {
+                label: 'Class name 2',
+                path: '/example',
+                aria_label: 'Link to Class name 2',
+              },
+            },
+          ],
+          list_class_name: 'ecl-footer-standardised__list--condensed',
+          content_before: 'More information on:',
+          section_class_name: 'ecl-footer-standardised__section--small-spacing',
         },
       ],
-      section_id: '6',
-      type: 'service_navigation',
-    },
-    {
-      links: [
-        {
-          link: {
-            label: 'Language policy',
-            path: '/example',
-            aria_label: 'Link to Language policy',
-          },
-        },
+    ],
+    [
+      [
         {
-          link: {
-            label: 'Cookies',
-            path: '/example',
-            aria_label: 'Link to Cookies',
+          title: {
+            link: {
+              label: 'European Commission',
+              path: 'https://ec.europa.eu/info/index_en',
+            },
           },
         },
+      ],
+      [
         {
-          link: {
-            label: 'Privacy policy',
-            path: '/example',
-            aria_label: 'Link to Privacy policy',
-          },
+          links: [
+            {
+              link: {
+                label: 'Contact the European Commission',
+                path: '/example',
+                aria_label: 'Link to Contact the European Commission',
+              },
+            },
+            {
+              link: {
+                label: 'Follow the European Commission on social media',
+                path: '/example',
+                aria_label:
+                  'Link to Follow the European Commission on social media',
+              },
+            },
+            {
+              link: {
+                label: 'Resources for partners',
+                path: '/example',
+              },
+            },
+          ],
+          section_class_name: 'ecl-footer-standardised__section--split-list',
```

Comment:
Same as the other comments related to these class names, not sure about the class name itself, it is kind of doing the opposite, like "joining" more than splitting, maybe, but here we might simply be able to select this section without any identifier, if we look for the last row, second column: ` .ecl-footer-standardised__row:last-child .ecl-footer-standardised__column:nth-child(2) .ecl-footer-standardised__section`

Reply:
This is questionable, I tried at the beginning to not use custom classes as possible on the sections by using nth-child(). the problem I see is if a row or columns is removed there could be an impact on the behavior of the grid which is not wanted… by using a specific class, the user will have the control?

Reply:
but clearly, I also want to remove these custom classes on the sections at markup level ... 
but using nth-child seems a bit tricky to me, both for the middle section and these last sections list

Reply:
well, basing on the usage page i'm now seeing that even the "class links" section is mandatory: https://ec.europa.eu/component-library/ec/standardised-template/footer/usage/, so the assumptions we could make are potentially even more...but ok, just we should not present this as a "use case", so the toggler for the class section should not be there.

Reply:
Ok I'm removing the toggler 👍  The class name is not mandatory for the harmonised footer, so I prefer to keep same class and css logic for both footer as they are the same.